udata/tests/test_activity.py

```diff
@@ -52,29 +52,30 @@ def test_create_and_retrieve_for_related(self):
         for activity in activities:
             self.assertIsInstance(activity, FakeActivity)
 
-    def check_emitted(self, sender, activity):
+    def callback(self, sender, activity):
         self.assertEqual(sender, FakeActivity)
         self.assertIsInstance(activity, FakeActivity)
-        self.emitted = True
+        self.callback_called = True
 
-    def test_emit_signal(self):
-        '''It should emit a signal on new activity'''
+    def test_callback_registration(self):
+        '''Check that the registered callback is executed when a new activity
```

Comment:
Test description are meant to be read on run in the console output. As a convention, we express what the tested code **should** do, not what the test itself checks (as test always checks something).
To keep the change, it could be:

     (it should) execute registered callbacks when a new activity is issued

(The `it` or `it should` are optionnals)